src/pages/docs/en/basics/local.md

```diff
@@ -0,0 +1,68 @@
+---
+title: Running locally
+section: Basics
+position: 3
+layout: "@docs"
+---
+
+## Starting a Development Server
+
+Once you've created a project, you can start developing your app locally.
+You can use the `space dev` command to start a local development server for your project.
+
+```bash
+space dev --open # start a local development server and open the app in your browser
+```
+
+You can control what command is run for each micro by editing the `dev` field in your `Spacefile`.
+
+```yaml
+v: 0
+micros:
+  - name: client
+    src: ./client
+    engine: svelte
+    primary: true
+    path: client
+  - name: api
+    src: ./api
+    engine: nodejs16
+    dev: node index.js
+    path: api
+```
+
```

Comment:
````suggestion

```bash
space dev --open # start a local development server and open the app in your browser
```

When running `space dev` the CLI will start its own web server on port `4200` that emulates the Deta Space runtime locally. It will start each Micro by running the `dev` command that is either set by the engine automatically or specified in the `Spacefile`. It automatically handles the routing between Micros and logs from every Micro are combined and sent to the console.

You can control what command is run for each Micro by editing the `dev` field in your `Spacefile`.

```yaml
v: 0
micros:
  - name: client
    src: ./client
    engine: svelte
    primary: true
    path: client
  - name: api
    src: ./api
    engine: nodejs16
    dev: node index.js
    path: api
```

The specified `dev` commands need to start a web server that listens on the port specified with the `PORT` environment variable. This way the CLI can properly forward requests to the Micro.

To develop and test your app open a browser and navigate to `http://localhost:4200`.

````